libs/execution_context/ebpf_native.c

```diff
@@ -54,6 +54,20 @@ typedef enum _ebpf_native_module_state
     MODULE_STATE_UNLOADING,
 } ebpf_native_module_state_t;
 
+typedef struct _ebpf_native_handle_cleanup_information
+{
+    size_t count_of_program_handles;
+    ebpf_handle_t* program_handles;
+    size_t count_of_map_handles;
+    ebpf_handle_t* map_handles;
+} ebpf_native_handle_cleanup_information_t;
+
+typedef struct _ebpf_native_handle_cleanup_context
+{
+    ebpf_native_handle_cleanup_information_t* handle_information;
+    ebpf_preemptible_work_item_t* handle_cleanup_workitem;
```

Comment:
```suggestion
    ebpf_preemptible_work_item_t* handle_cleanup_work_item;
```
nit: be consistent with type name